src/data/packer.json

```diff
@@ -1,12 +1,13 @@
 {
   "slug": "packer",
   "name": "Packer",
+  "basePaths": ["docs", "guides", "intro", "plugins"],
```

Comment:
Note that `guides` and `intro` are not shown in the top navigation, reflecting the current state on <https://www.packer.io>. As well, note that the `plugins` route is not yet built out - we'll add this to the UI assembly branch in a later, separate PR, as it's likely enough complexity to merit its own PR (see #47 and #118 for analogous work merged to `main`).

Reply:
All sounds good to me! 👍🏻